elixir/apps/web/lib/web/router.ex

```diff
@@ -140,6 +140,7 @@ defmodule Web.Router do
         end
 
         live "/:id/edit", Edit
+        live "/:id/groups", Groups
```

Comment:
```suggestion
        live "/:id/edit_groups", EditGroups
```

Reply:
I had considered this, but since we didn't have a `new`, `show`, or `index` page, I figured just having URL path be `groups` would be fine.  But, I'm not opposed to naming it this either.  I'll update it.

Reply:
As I was making this change, I realized that you had recommended changing the name of the module this is referencing to: `EditActorGroups`, but here you're recommending `EditGroups`.  Which of these two do you prefer?  I kind of like `EditGroups`, only because the module name will already have `Actors` in the name, e.g. `Web.Actors.EditGroups`, but I'm also ok, with going the other way too.

Reply:
I went ahead and used `EditGroups` for now.  If you feel strongly about it being `EditActorGroups`, just let me know and I'll get it updated.